static/js/details.js

```diff
@@ -147,8 +147,12 @@ $(function () {
         document.querySelectorAll("#log tr").forEach(function(row) {
             var dt = moment.unix(row.dataset.dt).utc();
             format == "local" ? dt.local() : dt.tz(format);
+            var dateFormat = "MMM D"
+            if (moment().year() != dt.format("YYYY")) {
+                dateFormat = "MMM D YYYY"
```

Comment:
IMO the date looks better with a comma separator before year – "MMM D, YYYY", could you add it?

Reply:
Sure, will do that